src/server/modules/user/resolvers.js

```diff
@@ -27,6 +27,14 @@ export default pubsub => ({
       }
     }
   },
+  User: {
```

Comment:
@mitjade Mitja, I wonder why this is needed? The resolvers seems to be defined as identity functions - they return their argument. I thought if they are absent - everything should work the same? So what is the catch?

Reply:
@vlasenko Not sure, but if I remove this `UserProfile` is not called.